pymc3/step_methods/sgmcmc.py

```diff
@@ -298,3 +311,98 @@ def competence(var, has_grad):
         if var.dtype in continuous_types and has_grad:
             return Competence.COMPATIBLE
         return Competence.INCOMPATIBLE
+
+
+class ConstantStochasticGradient(BaseStochasticGradient):
```

Comment:
maybe a shorter name?

Reply:
I can change it to CSG, just like what I did with SGFS. Is that okay ?

Reply:
Yeah, it's not great but for consistency probably the best option.

Reply:
I have renamed it